R/RLearner_fcregr_autoarima.R

```diff
@@ -0,0 +1,95 @@
+makeRLearner.fcregr.auto.arima = function() {
+  makeRLearnerForecastRegr(
+    cl = "fcregr.auto.arima",
+    package = "forecast",
+    par.set = makeParamSet(
+      makeIntegerLearnerParam(id = "d", lower = 0, upper = Inf),
+      makeIntegerLearnerParam(id = "D", lower = 0, upper = Inf),
+      makeIntegerLearnerParam(id = "max.p", lower = 0, upper = Inf, default = 5),
+      makeIntegerLearnerParam(id = "max.q", lower = 0, upper = Inf, default = 5),
+      makeIntegerLearnerParam(id = "max.P", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "max.Q", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "max.order", lower = 0, upper = Inf, default = 5),
+      makeIntegerLearnerParam(id = "max.d", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "max.D", lower = 0, upper = Inf, default = 1),
+      makeIntegerLearnerParam(id = "start.p", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "start.q", lower = 0, upper = Inf, default = 2),
+      makeIntegerLearnerParam(id = "start.P", lower = 0, upper = Inf, default = 1),
+      makeIntegerLearnerParam(id = "start.Q", lower = 0, upper = Inf, default = 1),
+      makeLogicalLearnerParam(id = "stationary", default = FALSE, tunable = TRUE),
+      makeLogicalLearnerParam(id = "seasonal", default = TRUE, tunable = TRUE),
+      makeDiscreteLearnerParam(id = "ic", values = c("aicc", "aic", "bic"), default = "aicc"),
+      makeLogicalLearnerParam(id = "stepwise", default = FALSE, tunable = TRUE),
+      makeLogicalLearnerParam(id = "trace", default = FALSE, tunable = FALSE),
+      makeLogicalLearnerParam(id = "approximation"),
+      makeIntegerLearnerParam(id = "truncate", lower = 1, upper = Inf),
+      makeDiscreteLearnerParam(id = "test", values = c("kpss","adf","pp"), default = "kpss"),
+      makeDiscreteLearnerParam(id = "seasonal.test", values = c("ocsb", "ch")),
+      makeLogicalLearnerParam(id = "allowdrift", default = TRUE, tunable = FALSE),
+      makeLogicalLearnerParam(id = "allowmean", default = TRUE, tunable = FALSE),
+      makeNumericLearnerParam(id = "lambda"),
```

Comment:
When the default is not listed, does it revert to the default of the model? The value is normally set to `NULL`